vertex/extract_metrics.py

```diff
@@ -52,21 +58,36 @@ def load_vertex_data(df, column, n_subjects, n_vertex, mask=None):
     else:
         return vertex_data, vertex_mean, vertex_std
 
-def save_mat(x,key,fname):
-    print("Saving ", np.shape(x), key, "to", fname)
-    scipy.io.savemat(fname, {'X': x})
+
+
+def save_matrix(x,key,fname,as_mat):
+    # If we want to save as .mat 
+    if as_mat:
+        fname = fname + '.mat'
+        print("Saving ", np.shape(x), key, "to", fname)
+        savemat(fname, {'X': x})
+
+    # Otherwise save as .npz
+    else:
+        fname = fname + '.npz'
+        print("Saving ", np.shape(x), key, "to", fname)
+        np.savez(fname, X=x)
+
+
+
 
 #LOAD CIVET MASK TO IDENTIFY MIDLINE/ CORPOS COLLOSUM REGION
 #IDENTIFY 'VALID VERTICES' - IE VERTICES NOT IN THIS REGION
 midline_mask = np.loadtxt(args.mask_file)
 valid_vertices = np.where(midline_mask==1)[0].tolist() # list of valid indices in civet .txt file
-invalid_vertices = np.where(midline_mask==0)[0].tolist() #list of invalid indices in civet .txt file
+#invalid_vertices = np.where(midline_mask==0)[0].tolist() #list of invalid indices in civet .txt file
```

Comment:
What's going on here?

Reply:
Why was it commented?

Reply:
Oh yeah, it isn't used anywhere in the script
Uncommented it for consistency